package.json

```diff
@@ -15,6 +15,7 @@
         "js-yaml": "3.13.1",
         "jsforce": "1.9.3",
         "json-stable-stringify": "1.0.1",
+        "runtime-plugin-manager-clone": "^0.1.0",
```

Comment:
@agarcia-sf I see this is hosted in your personal github org. Could you please move it to the vlocityinc so that all the dependencies for this that we've forked are in the same org.

Reply:
Yeah!, will do... this is temporary while I work on the PR to the original repo.